client/me/get-apps/download-card.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+
+const DownloadCard = ( { translate } ) => {
+	const trackAppDownloadClick = () => {};
+	const availableLinkTag = ( platformLink ) => {
+		return <a href={ platformLink } onClick={ trackAppDownloadClick } />;
+	};
+
+	const windowsLink = 'https://apps.wordpress.com/d/windows?ref=getapps';
+	const macLink = 'https://apps.wordpress.com/d/osx?ref=getapps';
+	const linuxTarLink = 'https://apps.wordpress.com/d/linux?ref=getapps';
+	const linuxDebLink = 'https://apps.wordpress.com/d/linux-deb?ref=getapps';
+
+	let buttonLink = windowsLink;
+	let cardTitle = translate( 'Desktop App for Windows' );
+	let requires = translate( 'Requires Windows Whatevs v12+. ' );
+	let translateComponents = {
+		firstAvailableLink: availableLinkTag( macLink ),
+		secondAvailableLink: availableLinkTag( linuxTarLink ),
+		thirdAvailableLink: availableLinkTag( linuxDebLink ),
+	};
+	let alsoAvailable = translate( 'Also available for: {{firstAvailableLink}}MacOS{{/firstAvailableLink}}, {{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, {{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.', { components: translateComponents } );
+
+	if ( navigator.platform && navigator.platform.length > 0 ) {
+		switch ( navigator.platform ) {
+			case 'MacIntel':
+				buttonLink = macLink;
+				cardTitle = translate( 'Desktop App for MacOS' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Desktop App' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).